### ISSUE TITLE
python@3.7.4: decompress error

### ISSUE THREAD

**Issue by djangobudo** [ScoopInstaller/Main#242]
[dark.log](https://github.com/ScoopInstaller/Main/files/3391379/dark.log)
Hi all,

WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Installing 'dark' (3.11.1) [64bit]
Starting download with aria2 ...
Download: [#dde7a9 64KiB/3.3MiB(1%) CN:1 DL:476KiB ETA:7s]
Download: [#dde7a9 480KiB/3.3MiB(13%) CN:1 DL:427KiB ETA:7s]
Download: [#dde7a9 1.1MiB/3.3MiB(34%) CN:1 DL:565KiB ETA:4s]
Download: [#dde7a9 2.1MiB/3.3MiB(63%) CN:1 DL:713KiB ETA:1s]
Download: [#dde7a9 3.0MiB/3.3MiB(89%) CN:1 DL:749KiB]
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: dde7a9|OK  |   762KiB/s|E:/nosave/scoop/cache/dark#3.11.1#https_raw.githubusercontent.com_ScoopInstaller_Binar
y_master_dark_dark-3.11.1.zip
Download: Status Legend:
Download: (OK):download completed.
Checking hash of dark-3.11.1.zip ... ok.
Extracting dark-3.11.1.zip ... done.
Linking E:\nosave\scoop\apps\dark\current => E:\nosave\scoop\apps\dark\3.11.1
Creating shim for 'dark'.
'dark' (3.11.1) was installed successfully!
Installing 'python' (3.7.4) [64bit]
Loading python-3.7.4-amd64.exe from cache.
Checking hash of python-3.7.4-amd64.exe ... ok.
Running installer script...
ERROR Exit code was 1!
Failed to extract files from E:\nosave\scoop\apps\python\3.7.4\python-3.7.4-amd64.exe.
Log file:
  E:\nosave\scoop\apps\python\3.7.4\dark.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/scoopinstaller/scoop-main/issues/new?title=python%403.7.4%3a+decompress+error

HTH

**Comment by niheaven** [comment]
Could you paste log file content here?

**Comment by niheaven** [comment]
I could not read the error message... Maybe it's French?

**Comment by djangobudo** [comment]
Sure.
Here it is.

python-3.7.4-amd64.exe
dark.exe : error DARK0001 : Impossible de cr?er un fichier d?j? existant.


Exception Type: System.IO.IOException

Stack Trace:
   ? System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   ? System.IO.__Error.WinIOError()
   ? System.IO.File.InternalMove(String sourceFileName, String destFileName, Boolean checkHost)
   ? Microsoft.Tools.WindowsInstallerXml.BurnReader.ExtractAttachedContainer(String outputDirectory, String tempDirectory)
   ? Microsoft.Tools.WindowsInstallerXml.Unbinder.UnbindBundle(String bundleFile, String exportBasePath)
   ? Microsoft.Tools.WindowsInstallerXml.Unbinder.Unbind(String file, OutputType outputType, String exportBasePath)
   ? Microsoft.Tools.WindowsInstallerXml.Tools.Dark.Run(String[] args)

> dark.exe : error DARK0001 : Impossible de cr?er un fichier d?j? existant.
It is indeed French.
Translation:
"dark.exe: error DARK0001 : Impossible to create an already existing file.

My apologies for the translation if it is not idiomatic.

Some element of context:

- I have indeed installed dark through scoop independently of python.
- Your comment rang a bell and I uninstalled dark and tried again to install version 3.7.4 of python.
- The installation was then successful.

HTH
